resources/models/resource.py

```diff
@@ -196,6 +196,14 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
         (PRICE_TYPE_WEEKLY, _('Weekly')),
         (PRICE_TYPE_FIXED, _('Fixed')),
     )
+    PLACEMENT_INSIDE = 'inside'
+    PLACEMENT_OUTSIDE = 'outside'
+    PLACEMENT_INSIDE_AND_OUTSIDE = 'both'
+    PLACEMENT_CHOICES = (
+        (PLACEMENT_INSIDE, _('Inside')),
+        (PLACEMENT_OUTSIDE, _('Outside')),
+        (PLACEMENT_INSIDE_AND_OUTSIDE, _('Inside and outside')),
+    )
```

Comment:
Would it be better to move this to a config/constants file

Reply:
I agree that it would be better to move it to some config file. However, this has been the pattern in this project, this is why I didn't want to deviate from it. May be it would be better if we move all the constants in some other pull request ?